tests/test_dataserver.py

```diff
@@ -1,8 +1,32 @@
+from pathlib import Path
 from hifz.dataserver import DataServer
 from hifz.models import Card
 
+def test_dataserver_returns_card_csv():
+    server = DataServer()
+    temp_file_path = "data/test_arabic.csv"
+    with Path(temp_file_path).open() as f:
+        f.writelines([
+            "front,back",
+            "ا,alif (a as in bad or A as in fAther)",
+            "ب,'baa"
```

Comment:
Thinks are backward since arabic if right to left, so the apostrophe appeared on the wrong side.
```suggestion
            "ب,baa'"
```